### PR TITLE
Feature/vv 144

### PR DESCRIPTION
**Pull request by fertandil87**
https://globalfishingwatch.atlassian.net/browse/VV-144

image

### CODE REVIEW

File: applications/vessel-history/src/features/vessels/vessels.utils.ts [hunk 1]
```diff
@@ -16,19 +16,30 @@ const getPriorityzedFieldValue = <T = any>(
...
     .filter(({ value }) => value !== null && value !== undefined)
     .map((dataValue) => ({
       value: dataValue.value,
       priority: fieldPriority.indexOf(dataValue.source),
     }))
-    .sort((a, b) => (a.priority > b.priority ? 1 : -1))
-    .map(({ value }) =>
+    .sort((a, b) => {
+      // If any of the values not exist, we use the other
+      if (!a.value || (Array.isArray(a.value) && !a.value.length)) {
```
Comment (rdgfuentes) [inline applications/vessel-history/src/features/vessels/vessels.utils.ts:28]: 👍

File: applications/vessel-history/src/features/vessels/sources/tmt.slice.ts [hunk 2]
```diff
@@ -16,6 +17,12 @@ const sortValuesByDate = (a: ValueItem, b: ValueItem) =>
 const extractValue: (valueItem: ValueItem[]) => string | undefined = (valueItem: ValueItem[]) => {
   return valueItem.slice().shift()?.value || undefined
 }
+const sortAuthorizations: (authorizations: Authorization[]) => Authorization[] = (authorizations: Authorization[]) => {
```
Comment (rdgfuentes) [inline applications/vessel-history/src/features/vessels/sources/tmt.slice.ts:20]: 👏

File: packages/api-types/src/vessel.ts [hunk 3]
```diff
@@ -17,7 +23,7 @@ export interface Vessel {
   owner?: string
   operator?: string
   builtYear?: string
-  authorizations?: string[]
+  authorizations?: Authorization[]
```
Comment (rdgfuentes) [inline packages/api-types/src/vessel.ts:26]: You'll have to publish a new version of this package

**Review by rdgfuentes — CHANGES_REQUESTED** [review]
Looks good! Before merging remember to generate the changeset to publish a new version of the api-types package

File: packages/api-types/src/vessel.ts (hunk 3 above)
Comment (fertandil87) [reply packages/api-types/src/vessel.ts:26]: yes

**Pull request merged**